src/spaceone/inventory/manager/workload/job_manager.py

```diff
@@ -0,0 +1,178 @@
+        """
+        collected_cloud_services = []
+        error_responses = []
+
+        secret_data = params["secret_data"]
+        pod_name = ""
+
+        ##################################
+        # 0. Gather All Related Resources
+        # List all information through connector
+        ##################################
+        pod_conn: JobConnector = self.locator.get_connector(
+            self.connector_name, **params
+        )
+        list_all_job = pod_conn.list_job()
+        list_all_pod = pod_conn.list_pod()
+
+        for job in list_all_job:
+            try:
+                ##################################
+                # 1. Set Basic Information
+                ##################################
+                job_name = job.metadata.name
+                cluster_name = self.get_cluster_name(secret_data)
+                region = "global"
+
+                ##################################
+                # 2. Make Base Data
+                ##################################
+                # key:value type data need to be processed separately
+                # Convert object to dict
+                raw_data = job.to_dict()
+                raw_readonly = job.to_dict()
+                raw_data["metadata"]["annotations"] = self.convert_labels_format(
+                    raw_data.get("metadata", {}).get("annotations", {})
+                )
+                raw_data["metadata"]["labels"] = self.convert_labels_format(
+                    raw_data.get("metadata", {}).get("labels", {})
+                )
+                raw_data["spec"]["node_selector"] = self.convert_labels_format(
+                    raw_data.get("spec", {}).get("node_selector", {})
+                )
+                raw_data["spec"]["selector"][
+                    "match_labels"
+                ] = self.convert_labels_format(
+                    raw_data.get("spec", {}).get("selector", {}).get("match_labels", {})
+                )
+
+                raw_data["spec"]["template"]["metadata"][
+                    "annotations"
+                ] = self.convert_labels_format(
+                    raw_readonly.get("spec", {})
+                    .get("template", {})
+                    .get("metadata", {})
+                    .get("annotations", {})
+                )
+                raw_data["spec"]["template"]["metadata"][
+                    "labels"
+                ] = self.convert_labels_format(
+                    raw_readonly.get("spec", {})
+                    .get("template", {})
+                    .get("metadata", {})
+                    .get("labels", {})
+                )
+                raw_data["spec"]["template"]["spec"][
+                    "node_selector"
+                ] = self.convert_labels_format(
+                    raw_readonly.get("spec", {})
+                    .get("template", {})
+                    .get("spec", {})
+                    .get("node_selector", {})
+                )
```

Comment:
I think this code could be modularized into a function,
for example, like this.
because i understand that this code is used to generate spec at `raw_data` 

```python
def _make_spec_data(self, spec):
    if node_selector := spec.get('node_selector'):
          spec['node_selector'] = self.convert_lables_format(node_selector)
    if ...
```